content/releases/3-6/changes/panel/link.txt

```diff
@@ -135,6 +135,22 @@ Fixes:
 
 Breaking:
 
+<info>
```

Comment:
Maybe we should move this repeating part to a snippet or a tag for easy integration into new release pages?